app/lib/simple_layout.rb

```diff
@@ -146,7 +146,6 @@ def create_builtin_pages_and_partials!
     create_builtin_partial!('signup/cas')
 
     # Messages
-    create_builtin_partial!('messages/menu')
```

Comment:
From what I can tell, this is not related to the deleted `app/views/provider/admin/messages/_menu.html.erb`, this is a template (partial "messages/menu") that is created in the developer portal CMS, so that the user can update it.

So, here and in other places I'll mark it as **revert**